server/src/processing/LinkDetectionModule/LinkDetectionModule.ts

```diff
@@ -33,6 +34,10 @@ export class LinkDetectionModule extends Module {
 
   public async main(doc: Document): Promise<Document> {
     let mdLinks = await this.extractLinksFromMetadata(doc.inputFile);
+    if (mdLinks === undefined) {
+      return doc;
+    }
```

Comment:
I will remove this coda if we do the update commented below.

Reply:
The issue #212 is solved by PR #214 ?

Reply:
> The issue #212 is solved by PR #214 ?

Yes